content/influxdb/v2.0/reference/rc0-upgrade-guide.md

```diff
@@ -0,0 +1,271 @@
+we had to make breaking changes to the underlying storage engine for InfluxDB 2.0.
+
+{{% note %}}
+If you have questions upgrading please
+[open an issue](https://github.com/influxdata/influxdb/issues)
+or join the [Community Slack workspace](https://influxcommunity.slack.com/) to get immediate help.
+{{% /note %}}
+
+## 1. Disable existing integrations
+
+To begin, if you have any software interacting with your InfluxDB beta instance, pause those integrations until the upgrade is complete.
+This includes Telegraf, client libraries, and any custom applications you may have developed.
+Basically, anything reading, writing, or monitoring your InfluxDB instance should be shut off and re-enabled after the upgrade.
+
+## 2. Stop existing InfluxDB beta instance
+
+Next, shut down your existing InfluxDB beta instance.
+You can manually kill the individual process using **Control+c**
+(or by finding the process ID with `ps aux | grep -i influxd` and using `sudo kill -9 <PID>`).
+If you've set `influxd` to run as a system process, follow the same steps you would use to disable any system process.
+
+## 3. (Optional) Rename existing InfluxDB binaries
+
+To easily identify your existing InfluxDB binaries, rename them `influx_old` and `influxd_old`.
+This is helpful if you've installed the binaries in your path.
+We use the names `influxd_old` for this guide, but you can use whatever you like.
+
+## 4. Move existing data and start the latest InfluxDB
+
+If you have not already, [download the InfluxDB OSS 2.0rc0](https://portal.influxdata.com/downloads/).
+
+To move data between the two instances, first configure both the old and new instances of InfluxDB to run at the same time.
+If you download the latest InfluxDB beta, and try to start it up with existing data, it will most likely refuse to start.
+You will see an error message in the terminal this old data isn't compatible.
+
+```
+Incompatible InfluxDB 2.0 version found.
+Move all files outside of engine_path before influxd will start.
+```
+
+So let's listen to the message and move our previous data to a safe location:
+
+```sh
+mv ~/.influxdbv2 ~/.influxdbv2_old
+```
+
+(You can move it to wherever you'd like.)
+When we start the old instance again, we will tell it where your data files are located.
+<!-- we'll provide this location -->
+
+Start the latest InfluxDB version by running
+
+```sh
+influxd
+```
+
+Since the data folder has been moved, everything will be empty.
+<!-- You can check out http://localhost:8086 in your browser and see a setup page, but don't set it up yet. -->
+You can check out http://localhost:8086 in your browser and see a setup page, but don't go through the setup process yet.
+
+## 5. Start old InfluxDB beta instance
+
+You can now start your old influxdb instance and point it to your old data directory:
+
+```sh
+influxd_old --bolt-path ~/.influxdbv2_old/influxd.bolt --engine-path ~/.influxdbv2_old/engine
+```
+
+If you used a different location, update as appropriate.
```

Comment:
it might also be worth noting that the port numbers have changed from 9999 to 8086 in the latest version, so there should be no conflict when running both instances. but that might be more detail than whats needed here.

Reply:
Thanks for flagging this. Let's see what some other readers think when we have them test this guide.